packages/vscode-extension/src/webview/Preview/MjpegImg.tsx

```diff
@@ -61,7 +68,14 @@ const MjpegImg = forwardRef<HTMLImageElement, React.ImgHTMLAttributes<HTMLImageE
       };
     }, [ref]);
 
-    return <img ref={ref} {...rest} />;
+    function handleError(e: React.SyntheticEvent<HTMLImageElement, Event>) {
+      const img = e.currentTarget;
+      if (img.src !== NO_IMAGE_DATA) {
+        img.src = NO_IMAGE_DATA;
+      }
```

Comment:
NIT: You could just 
```suggestion
        img.src = NO_IMAGE_DATA;
```
it should be noop if the condition isn't met.